packages/web3-eth/src/types.ts

```diff
@@ -262,22 +255,27 @@ export type SendTransactionEvents = {
 	sending: Transaction;
 	sent: Transaction;
 	transactionHash: HexString32Bytes;
-	receipt: ReceiptInfo;
+	receipt: ReceiptInfoFormatted;
 	confirmation: {
 		confirmationNumber: number;
-		receipt: ReceiptInfo;
+		receipt: ReceiptInfoFormatted;
 		latestBlockHash: HexString32Bytes;
 	};
 };
 
+export interface SendTransactionOptions {
+	returnType?: ValidTypes;
+	ignoreGasPricing?: boolean;
+}
+
 export type SendSignedTransactionEvents = {
```

Comment:
There is same promise combined event emitter for `sendTransaction` and `sendSignedTransaction`
```suggestion
export type SendTransactionEvents = {
```
could you make these rename changes in other parts of code as well.

Reply:
[Updated](https://github.com/ChainSafe/web3.js/blob/6c6d829a53e7e46e85a1784b894b756ec6e07cf4/packages/web3-eth/src/types.ts#L271), but `SendSignedTransactionEvents` emits `HexStringBytes` for both `sending` and `sent` as opposed to a full `Transaction` object in `SendTransactionEvents